src/main/java/codeu/model/data/User.java

```diff
@@ -27,6 +28,7 @@
   private long messageCount;
   private boolean isAdmin;
   private  String description;
+  private List<String> defaultAdminList;
```

Comment:
Just read your other comment on not adding Guava for one line. I think thats fair, but guava is pretty useful in alot of cases so if there is any resource that will probably come up again it is Guava. It may be one line now but it will probably turn into more in the future.

Also, I try to make my comments so you try things another way or learn something new. Idk if you've added dependencies to maven projects in the past but I think it'll be good practice to do so and it cleans up the code. There is an argument that this takes up space but we have plenty of that, one more dependency isnt going to hurt.

Reply:
Kirielle just had a use case for Guava as well. Same Lists API.

Reply:
done